.gitignore

```diff
@@ -83,6 +83,9 @@ ipython_config.py
 # PEP 582; used by e.g. github.com/David-OConnor/pyflow
 __pypackages__/
 
+# PyCharm
+/.idea/
```

Comment:
Looks like their ideal fix is to use https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore rather than language-specific gitignores - no other language ignores .idea